DEPLOYING.md

```diff
@@ -0,0 +1,38 @@
+# Deploying CAMI Cloud
+CAMI Cloud infrastructure is made of Docker Containers that are run from images. The images that are used for production deployment are stored and pulled from [Docker Hub](https://hub.docker.com/r/vitaminsoftware/cami-project/). Whenever the `master` branch of the project is modified, the images are automatically rebuilt by Docker Hub.
+
+Let's assume that you just pushed some commits in the `master` branch. The steps that must be followed afterwards are described below.
+
+## Simple procedure
+
+### 1. Check the build status of the Docker Images
+ * Go to [Docker Hub](https://hub.docker.com/r/vitaminsoftware/cami-project/) to see the build status of the project's Docker Images
+* Wait for them to be successfully built
+
+### 2. Upgrade the containers from Rancher Web UI
+* Go to [CAMI's Rancher Web UI](http://138.68.92.229:8080/) and select the **CAMI** stack. There you'll find the already set-up docker containers
+* Select the ones you know were modified and click the **Upgrade** button (up arrow icon) from the right of the entry row
```

Comment:
@cretualexandru: aren't you supposed to first stop the containers before upgrading?